core/src/main/kotlin/Kord.kt

```diff
@@ -371,6 +370,10 @@ class Kord(
         return defaultSupplier.getGuildApplicationCommands(resources.applicationId, guildId)
     }
 
+    suspend fun getOriginalInteractionResponse(token: String): Message? {
+        return EntitySupplyStrategy.rest.supply(this).getOriginalInteractionOrNull(resources.applicationId, token)
+    }
```

Comment:
What's the motivation of making this a 'top-level' Kord function? Seems like it belongs to an interaction behavior instead.